doc/reference/config-options-launch.rst

```diff
@@ -0,0 +1,103 @@
+.. _launch-config-options:
+
+Launch Configuration Options
+##############################
+
+As explained in :ref:`acrn_configuration_tool`, launch configuration files
+define post-launched User VM settings. This document describes these option settings.
+
+``uos``:
+  Specify the User VM with its relative ID to Service VM by the ``id``
```

Comment:
Should this be "_the_ Service VM"? Also, probably "_using_ the ``id`` attribute".

Reply:
> Should this be "_the_ Service VM"? Also, probably "_using_ the `id` attribute".

I'm not sure if it's "using" or maybe "based on"? @dbkinder Do Ben's edits look OK?

Reply:
That is a awkward sentence isn't it.  Keeping a parallel structure the subsequent descriptions, how about this:
```
Specify the User VM ``id`` to the Service VM.
```